397 Add doc for anonymous usage stats (#4104)
grafana/pyroscope: docs/sources/get-started/_index.md

Verify that you have installed [Docker](https://docs.docker.com/engine/install/).

{{< admonition type="note" >}}

> Pyroscope includes a system that optionally and anonymously reports non-sensitive, non-personally identifiable information about the running Pyroscope cluster to a remote statistics server to help Pyroscope maintainers understand how the open source community runs Pyroscope.

Author: This is the text we have in Tempo. Including it as a comparison. it has a gentler tone to it: 
> By default, Tempo reports anonymous usage data about the shape of a deployment to Grafana Labs. This data is used to determine how common the deployment of certain features are, if a feature flag has been enabled, and which replication factor or compression levels are used.
> By providing information on how people use Tempo, usage reporting helps the Tempo team decide where to focus their development and documentation efforts. No private information is collected, and all reports are completely anonymous. Reporting is controlled by a configuration option.

That text is too long for an admonition. Maybe: 

Pyroscope reports anonymous usage data about the shape of a deployment to Grafana Labs. By providing information on how people use Tempo, usage reporting helps the Tempo team decide where to focus their development and documentation efforts. No private information is collected, and all reports are completely anonymous. Reporting is controlled by a configuration option, refer to [docs link] for information.

Reviewer: ```suggestion
By default, Pyroscope reports anonymous non-sensitive, non-personally identifiable information about the running cluster to a remote statistics server to help Pyroscope maintainers understand how the open source community runs Pyroscope.
```
